R/makeMicrobiomeExperimentFromphyloseq.R

```diff
@@ -17,6 +17,12 @@
 #'     makeMicrobiomeExperimentFromphyloseq(enterotype)
 #'     data(esophagus, package="phyloseq")
 #'     makeMicrobiomeExperimentFromphyloseq(esophagus)
+#'
```

Comment:
Examples should be run. What is the intent here? The fix should be applied inside `makeMicrobiomeExperimentFromphyloseq`. Please suggest a general fix for this function